### PR TITLE
add ng-table v0.8.3 and update package.json

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/esvit/ng-table
**Watch 133
  Star 1,748
 Fork 553**
@LeaYeh could u help me check this PR for #5002? thank you!
- [ ] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
[IMPORTANT - No more manual submissions](https://github.com/cdnjs/cdnjs#important---no-more-manual-submissions)

**Comment by maruilian11** [comment]
@PeterDaveHello  you mean i should just update the git update field?

**Comment by PeterDaveHello** [comment]
You don't need to update it with new version.

**Comment by maruilian11** [comment]
@PeterDaveHello but how can the auto-update run with wrong basepath?

**Comment by PeterDaveHello** [comment]
I didn't say that you can't update its auto-update config.

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR ? thank you very much!

File: ajax/libs/ng-table/package.json [hunk 1]
```diff
@@ -8,16 +8,19 @@
...
+    "basePath": "dist",
     "files": [
-      "ng-table.js",
-      "ng-table.css",
-      "ng-table.min.js",
-      "ng-table.min.css",
-      "ng-table.map"
+      "ng-table*.js",
+      "ng-table*.css",
+      "ng-table*.map"
```
Comment (Amomo) [inline ajax/libs/ng-table/package.json]: I think we could use `ng-table*.+(js|css|map)` here

**Comment by Amomo** [comment]
@maruilian11 could you separate your commit to "add keywords ..." and "add auto-update ..." ?

**Comment by maruilian11** [comment]
@Amomo thank you for your reminding!!! i have corrected it all~
could you help me check this PR again? thank you very much!!

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look?
Thank you~ :-D

**Comment by PeterDaveHello** [comment]
Should also update PR's title next time.

**Pull request merged**